addons/core/translations/resources/en-us.yaml

```
@@ -278,6 +278,9 @@ target:
   types:
     tcp: TCP
     ssh: SSH
+  help: 
+    tcp: This target belongs to an SSH host catalog and will use SSH to connect
+    ssh: This target belongs to an TCP host catalog and will use TCP to connect
```

Should these be flipped?

```suggestion
  help: 
    tcp: This target belongs to an TCP host catalog and will use TCP to connect
    ssh: This target belongs to an SSH host catalog and will use SSH to connect
```

---

ah yes!